### DISCUSSION TITLE
Setting to automatically re-slice when changing settings?

### DISCUSSION THREAD

**Discussion by MagnusThome** [OrcaSlicer/OrcaSlicer#941] [General]
Just wondering if there is a setting somewhere to automatically slice so I don't have to manually push the slice button each time I change a slice/print setting?

Thanks for the great work on this!

**Comment by Pileot** [comment]
Been almost a year, has this ever been looked at?

**Comment by torndar** [comment]
Looks like was actually done in https://github.com/OrcaSlicer/OrcaSlicer/pull/11407
